src/lib/plot/axis/axis.js

```diff
@@ -140,18 +145,20 @@ class Axis extends PureRenderComponent {
     };
 
     const {
-      left,
-      top,
-      width,
+      className,
       height,
+      left,
       orientation,
-      title
+      title,
+      top,
+      width
     } = props;
-
+    const isY = [LEFT, RIGHT].indexOf(orientation) > -1;
```

Comment:
`isY` as a variable name might be a little confusing since there's an attribute and it may not be `y`